<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PtzAdvancedPage</class>
 <widget class="QWidget" name="PtzAdvancedPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_channel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Channel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ComboBox" name="comboBox_channel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_connectionType">
       <property name="text">
        <string>Connection Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ComboBox" name="comboBox_connectionType"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_baudRate">
       <property name="text">
        <string>Baud Rate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ComboBox" name="comboBox_baudRate"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_dataBit">
       <property name="text">
        <string>Data Bit</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ComboBox" name="comboBox_dataBit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_stopBit">
       <property name="text">
        <string>StopBit</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="ComboBox" name="comboBox_stopBit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_checksumBit">
       <property name="text">
        <string>Checksum Bit</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="ComboBox" name="comboBox_checksumBit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_protocol">
       <property name="text">
        <string>Protocol</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="ComboBox" name="comboBox_protocol"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_address">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="ComboBox" name="comboBox_address"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>301</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_copy">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_back">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
